### DISCUSSION TITLE
Docs: Error on creating new branch, unclear steps on documentation

### DISCUSSION THREAD

**Discussion by maurobusso** [vercel/next.js#48591] [Help]
### What is the improvement or update you wish to see?

Creation of new branch.

### Is there any context that might help us understand?

I was looking to contribute to this awesome project, but after having followed the first few steps on the developing docs:

1. Install the GitHub CLI

2. Clone the Next.js repository with this command

gh repo clone vercel/next.js -- --depth=3000 --branch canary --single-branch

3. Create a new branch with this command

git checkout -b MY_BRANCH_NAME origin/canary

I get the following error: 

![nextjs-issue](https://user-images.githubusercontent.com/107254152/233136699-b0b3f8f3-a1b2-46d9-94eb-17220b9a56a4.png)

I might just be me, but perhaps is something unclear in the docs. It could be useful especially for new people that want to start contributing. :)

### Does the docs page already exist? Please link to it.

https://github.com/vercel/next.js/blob/canary/contributing/core/developing.md

**Comment by balazsorban44** [comment]
Hi, you might have to `git fetch --all` to make sure the canary branch is visible by git.

FWIW, it worked for me when following the contributing document, so it might be some local issue.

**Comment by maurobusso** [comment]
Hi, all right thank you. 

Yes it works now. I think it was just my mistake sorry.

Thanks a lot!
